### PR TITLE
Fix teams stats

### LINKED ISSUE #10969: Incorrect rewards shhowing in mainnet teams leadeboard
Gnilrad has rewards ad has games but showuldnt 

image

I suspect these are the LP rewards which are individual which I'm digging into.

If I am right then the teamStats API is not filtering on entityScorpre = Team as it should.

### PR DESCRIPTION
**Pull request by ValentinTrinque**
Close #10969

### CODE REVIEW

File: datanode/sqlstore/migrations/0076_teams_stats.sql [hunk 1]
```diff
@@ -1,71 +1,83 @@
...
 CREATE TABLE IF NOT EXISTS teams_stats
 (
-  team_id              BYTEA             NOT NULL,
-  party_id             BYTEA             NOT NULL,
-  at_epoch             BIGINT            NOT NULL,
-  total_quantum_volume HUGEINT           NOT NULL,
-  total_quantum_reward NUMERIC(1000, 16) NOT NULL,
-  games_played         JSONB             NOT NULL,
-  PRIMARY KEY (team_id, party_id, at_epoch)
+    team_id              BYTEA             NOT NULL,
```
Comment (EVODelavega) [inline datanode/sqlstore/migrations/0076_teams_stats.sql:4]: You can't update a migration that has already been deployed

File: datanode/sqlstore/accounts.go [hunk 2]
```diff
@@ -186,7 +186,7 @@ func (as *Accounts) getAccountFromCache(id entities.AccountID) (entities.Account
 	return entities.Account{}, false
 }
 
-func deterministicIDFromAccount(a *entities.Account) entities.AccountID {
+func DeterministicIDFromAccount(a *entities.Account) entities.AccountID {
```
Comment (EVODelavega) [inline datanode/sqlstore/accounts.go:189]: why export this func?

Comment (ValentinTrinque) [reply datanode/sqlstore/accounts.go:189]: For test purpose, so I can craft the account ID for the transfers foreign key.

File: datanode/sqlstore/migrations/0076_teams_stats.sql (hunk 1 above)
Comment (ValentinTrinque) [reply datanode/sqlstore/migrations/0076_teams_stats.sql:4]: Indeed, that's a mistake.

**Review by EVODelavega — APPROVED** [review]

**Pull request merged**